gamestate.py

```diff
@@ -652,32 +742,32 @@ def update_used_zpower(self, pokemon):
 					print(f'{GameState.Player.two}: ' 
 						f'Pokemon {name} was unexpectedly alive')
 
-	def check_moves(player, pokemon, move_names):
-		has_moves = gs.check_moves(player, pokemon)
-		if set(has_moves) != set(move_names):
-			print(f'{pokemon} has moves {has_moves} instead of {move_names}')
+	# def check_moves(player, pokemon, move_names):
```

Comment:
why is this stuff commented out?

Reply:
So the thing is, previously set_moves() took move_name as input and would set that move as 1.0, meaning it's a valid move for pokemon. I now changed set_move() to take pp, max_pp as well for input (we get pp, max_pp from request json) so now, set_move() will the given move_name's value to be pp/max_pp. Intuition is, all valid moves will start with 1.0 as pp=max_pp but as a move is getting used more and more, it's value will decrease till it reaches 0.0 at which point it's not a valid move. So the test cases don't work for this new definition of set_move(). I wrote in the PR comment that I didn't change the test case because in the next PR, i'll be including expansive test cases.

Reply:
resolved in the next commit